### PR TITLE
Number of K-Length Sublists with Average at Least Target [Python]

### PR DESCRIPTION
**Pull request by kushal13112001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have updated the Number of K-Length Sublists with Average at Least Target [Python] #2545 file

### Checklist

- [✔️] I've read the contribution guidelines.
- [✔️] I've checked the issue list before deciding what to submit.
- [✔️] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2545

### CODE REVIEW

**Comment by kushal13112001** [comment]
sir @bhav09 i have added the file please verify and merge

**Comment by kushal13112001** [comment]
sir @HarshCasper please add Gssoc'21 label

**Comment by kushal13112001** [comment]
@bhav09 sir please review my PR ....

File: Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py [hunk 1]
```diff
@@ -28,11 +28,14 @@
 if (p//k)>=target: #verifying the taken avg sum greater than target or not
```
Comment (bhav09) [inline Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py]: You have been doing great! However, a small correction can be made. Always make sure to add comments above the code segment and never alongside. Can you please correct it and request it again with all the modifications. That would be appreciated :)

**Review by bhav09 — CHANGES_REQUESTED** [review]
Always make sure to add comments above the code segment and never alongside. Can you please correct it and request it again with all the modifications. That would be appreciated :)

**Comment by kushal13112001** [comment]
yes @bhav09 sir sure

File: Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py (hunk 1 above)
Comment (kushal13112001) [reply Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py]: @bhav09 sir changes are done

**Comment by kushal13112001** [comment]
@bhav09 sir please review i have made the required changes

File: Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# Constraints:-
+
+# n ≤ 100,000 where n is the length of nums
+
+# Approach : First i will take the sum of [0:k] elements and check it's avg.
+# If avg greater than target ans count is increased
+# then in the loop starting from kth index adding the new element 
+# and removeing the initial added elements simultaneously
+
+
```
Comment (bhav09) [inline Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py:15]: Great work! Make a small change by mentioning the time complexity at last; after the last line of code. 
Happy learning!

Comment (kushal13112001) [reply Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py:15]: @bhav09 sir after approach i have mentioned ? should i change?

Comment (bhav09) [reply Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py:15]: Just make sure the complexity and I/O is written after the code.

Comment (kushal13112001) [reply Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py:15]: ok sir

Comment (kushal13112001) [reply Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py:15]: @bhav09 sir as you said changes have been made to the file, please review sir

**Comment by kushal13112001** [comment]
@HarshCasper and  @bhav09 sir all changes are done

**Comment by kushal13112001** [comment]
Tq @bhav09 and  @HarshCasper sir for starting my Gssoc'21 Journey by merging my 1st PR, I wish to contribute more and more to this Repository......

**Comment by kushal13112001** [comment]
Sir @bhav09 @HarshCasper please add level to my PR...

**Comment by kushal13112001** [comment]
sir @bhav09 Level-3 label got removed i guess........ that is y in profile section my score is showing 0.So please can you look  into it

**Comment by HarshCasper** [comment]
@all-contributors bot please add @kushal13112001 for code

**Comment by kushal13112001** [comment]
Tq @HarshCasper now got the score in profile

**Pull request merged**